merkletree/pad.go

```diff
@@ -76,48 +93,65 @@ func (pad *PAD) Update(policies Policies) {
 		pad.loadedEpochs = append(pad.loadedEpochs[:0], pad.loadedEpochs[n:]...)
 	}
 
-	pad.updateInternal(policies, pad.currentSTR.epoch+1)
+	pad.updateInternal(policies, pad.latestSTR.epoch+1)
 }
 
 func (pad *PAD) Set(key string, value []byte) error {
-	return pad.tree.Set(key, value)
+	str := pad.latestSTR
+	if str == nil {
```

Comment:
Good question. I saw there was a check for that after calling `Update()`here https://github.com/coniks-sys/coniks-go/pull/19/files/e82f666f1930a475550a8c3eb00194932c003a19#diff-475ca4a45242c730e6134d970fe64530L46 and wanted to be consistent here. If no combination of calling public methods can lead to `pad.latestSTR == nil` I'll remove it.